modules/nf-core/mirtop/counts/environment.yml

```diff
@@ -4,10 +4,11 @@ channels:
   - conda-forge
   - bioconda
 dependencies:
-  - bioconda::mirtop=0.4.28
-  - bioconda::pybedtools=0.10.0
-  - bioconda::pysam=0.22.1
+  - bioconda::mirtop=0.4.30
   - bioconda::samtools=1.21
+  - bioconda::pysam=0.22.1
```

Comment:
why the change in order? I think you would make @maxulysse happy if you could keep them in alphabetical order

Reply:
Please address this comment